useful-scripts/get_data_block_responses/get_data_block_responses.py

```diff
@@ -105,72 +120,63 @@ def print_to_console(text):
         url = f'{data_api_url}/tablebuilder/release/{releaseId}'
 
     if args.stage == "filters":
+        url = f'{data_api_url}/meta/subject'
         queryDict.pop("Filters")
         queryDict.pop("Indicators")
-        url = f'{data_api_url}/meta/subject'
 
     if args.stage == "time_periods":
+        url = f'{data_api_url}/meta/subject'
         queryDict.pop("Filters")
         queryDict.pop("Indicators")
         queryDict.pop("TimePeriod")
-        url = f'{data_api_url}/meta/subject'
 
-    query = json.dumps(queryDict, sort_keys=True, indent=2)
-
-    assert url is not None
-    assert query is not None
-
-    if os.path.exists(f'{results_dir}/block_{guid}'):
-        continue
-
-    headers = {
-        'Content-Type': 'application/json'
-    }
+    def write_this_block(status_code, response_time, response_dict):
+        write_block_to_file(
+            block_id=blockId,
+            release_id=releaseId,
+            subject_id=subjectId,
+            request_dict=queryDict,
```

Comment:
Should queryDict instead be query_dict to fit in with the other variable naming conventions?